frontend/src/libs/packages/api/base-http-api.ts

```diff
@@ -106,6 +110,11 @@ class BaseHttpApi implements HTTPApi {
       };
     }
 
+    if (response.status === HTTPCode.UNAUTHORIZED) {
+      await this.storage.drop(StorageKey.TOKEN);
+      window.location.assign(AppRoute.SIGN_IN);
```

Comment:
Not sure that is needed for us

Reply:
@v0ldemar01  Where do you want me to sign out user then?🤔

Reply:
we need to handle this in handleError middleware

Reply:
Maybe I am getting smth wrong, but I can't get error code from actions in the handleError middleware. It's working with signOut and this way error msg pop ups too, but thought that comparing with 401 would be correct